src/test/scala/query/AssignmentSpec.scala

```diff
@@ -0,0 +1,80 @@
+package com.originate.scalypher.test.query
+
+import com.originate.scalypher.types.ReferenceableMap
+import com.originate.scalypher.Property
+import com.originate.scalypher.Label
+import com.originate.scalypher.path.AnyNode
+import com.originate.scalypher.path.AnyRelationship
+import com.originate.scalypher.RemovePropertyAssignment
+import com.originate.scalypher.MergePropertiesAssignment
+import com.originate.scalypher.OverwriteAssignment
+import com.originate.scalypher.PropertyAssignment
+
+import org.scalatest._
+
+class AssignmentSpec extends WordSpec with Matchers {
+
+  val startNode = AnyNode()
+  val endNode = AnyNode()
+  val relationship = AnyRelationship()
+
+  val label1 = Label("label1")
+  val label2 = Label("label2")
+  val properties = Seq(
+    Property("name", "matt"),
```

Comment:
oh, sorry i didn't see that the properties were setting the same attribute. well, it's valid neo4j to write `match (n) set n.name = "a", n.name = "b" return n limit 1` and neo4j does what you'd expect, so n.name becomes "b". you think i need a test for that though?